practice5/src/test/java/com/academiaDigital/softwareArchitecture/practice5/parts/EngineTest.java

```diff
@@ -0,0 +1,29 @@
+package com.academiaDigital.softwareArchitecture.practice5.parts;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class EngineTest {
+
+    @Test
+    public void getVolume_should_return_volume() {

```

Comment:
you should maintain consistency with the names of the methods, why are you using camelCase on some of them and snake case on these ones?

Reply:
you are right, I changed that